attribution-graphs-frontend/attribution_graph/util-cg.js

```
@@ -0,0 +1,639 @@
+    // add ppClerp
+    await Promise.all(nodes.map(async d => {
+      if (!d.clerp) d.clerp = ''
+      d.remoteClerp = ''
+    }))
+
+    // condense nodes into features, using last occurence of feature if necessary to point to a node
+    var features = d3.nestBy(nodes.filter(d => d.isFeature), d => d.featureId)
+      .map(d => ({
+        featureId: d[0].featureId,
+        feature_type: d[0].feature_type,
+        clerp: d[0].clerp,
```

Biome (1.9.4)

[error] This property value named feature_type is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named feature_type


(lint/suspicious/noDuplicateObjectKeys)